assets/app/view/create_game.rb

```diff
@@ -95,7 +95,11 @@ def submit
         players = params
           .select { |k, _| k.start_with?('player_') }
           .values
-          .map { |name| { name: name } }
+          .map { |name| name.gsub(/\s+/, ' ').strip }
```

Comment:
ehhh i don't think it's necessary to do this since the names are user input

Reply:
If our stance is that users should avoid Hotseat usernames that result in indistiguishable display strings, we should just close #71 as won't fix.

Reply:
well the difference is that white space is only confusing whereas duplicate names breaks the game

Reply:
You're right. I was thinking that having two indistinguishable players also effectively breaks the game for a Hotseat play.

I usually try to avoid solving a bigger problem than the one described in the issue. I'll be extra careful about that in future PRs. 👍